src/main/kotlin/step3/domain/car/action/MovingStrategyImpl.kt

```diff
@@ -0,0 +1,14 @@
+package step3.domain.car.action
+
+import step3.util.Utils
+
+private const val MOVING_MINIMUM_CONDITION_NUMBER = 4
+
+class MovingStrategyImpl : MovingStrategy {
+    override fun isMovable(): Boolean {
+        if (MOVING_MINIMUM_CONDITION_NUMBER <= Utils.generateRandomNumber(Utils.RANDOM_NUMBER_START_INDEX, Utils.RANDOM_NUMBER_END_INDEX)) {
+            return true
+        }
+        return false
```

Comment:
```suggestion
        return if (MOVING_MINIMUM_CONDITION_NUMBER <= Utils.generateRandomNumber(Utils.RANDOM_NUMBER_START_INDEX, Utils.RANDOM_NUMBER_END_INDEX)) {
           true
        } else {
           false
        }
```

코틀린에서 if문은 표현식으로도 사용되니 이렇게 표현할수도 있겠네요 😄

Reply:
@sah3122 리뷰어님, 조언 감사합니다 😄 

한가지 궁금한 점이 있는데요. 조언해 주신 대로 하는 것은 [객체 지향 생활 체조 원칙](https://edu.nextstep.camp/s/56sRKVkV/ls/6RSdcCIN)의 `2. else 예약어를 쓰지 않는다.` 와는 무관한 것인가요?

Reply:
@sah3122 @dean-wev 리뷰어님,

해당 기능이 람다식으로 리팩토링되어, 조언해 주신 방법대로 수정을 못하게 되었습니다 😭 
알려주신 내용은 꼭 기억해 두고 추후에 사용하겠습니다 😄 

- https://github.com/next-step/kotlin-racingcar/commit/c10017fe3af127ef008596ab815037d7a11805db